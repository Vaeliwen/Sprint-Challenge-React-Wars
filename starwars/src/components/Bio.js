import React from 'react'

export const Bio = (props) => {
    return(
        <div>
            <h2>{props.char.name}</h2>
            <p>Height: {props.char.height} Weight: {props.char.mass}</p>
            <p>Hair: {props.char.hair_color} Skin: {props.char.skin_color}</p>
            <p>Eyes: {props.char.eye_color} YOB: {props.char.birth_year}</p>
            <p>Gender: {props.char.gender}</p>
        </div>
    )
}