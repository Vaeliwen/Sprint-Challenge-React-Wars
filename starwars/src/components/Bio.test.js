import React from 'react';
import renderer from 'react-test-renderer';

import {Bio} from './Bio'
import { Item } from 'semantic-ui-react';

describe('<Bio />', () => {
    it('Should match snapshot', () => {
        const tree = renderer.create(<Bio char='{
            "name": "Luke Skywalker",
            "height": "172",
            "mass": "77",
            "hair_color": "blond",
            "skin_color": "fair",
            "eye_color": "blue",
            "birth_year": "19BBY",
            "gender": "male"}'/>)
            expect(tree).toMatchSnapshot();
    })
})