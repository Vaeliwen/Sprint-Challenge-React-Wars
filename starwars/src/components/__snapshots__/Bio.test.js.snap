// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Bio /> Should match snapshot 1`] = `
<div
  className="ui text container"
  style={
    Object {
      "padding": "10px",
    }
  }
>
  <h2
    className="ui header"
  />
  <p>
    Height: 
     Weight: 
  </p>
  <p>
    Hair: 
     Skin: 
  </p>
  <p>
    Eyes: 
     YOB: 
  </p>
  <p>
    Gender: 
  </p>
</div>
`;
